index.html

```diff
@@ -15,50 +15,7 @@
 <body>
   <header class="header">
     <div class="container-nav">
-      <svg display="none">
-        <symbol viewBox="0 0 164.000000 38.000000" id="logo">
-          <g transform="translate(0.000000,38.000000) scale(0.100000,-0.100000)" fill="rgba(255,255,255,0.9)"
-            stroke="none">
-            <path d="M68 323 c9 -3 12 -38 12 -123 0 -85 -3 -120 -12 -123 -7 -3 31 -6 85
--6 l97 -1 0 31 c0 35 -23 36 -28 1 -3 -18 -10 -22 -43 -22 l-39 0 0 119 c0 86
-3 121 13 124 6 3 -13 5 -43 5 -30 0 -49 -2 -42 -5z" />
-            <path d="M368 296 c-16 -32 -15 -36 7 -82 l24 -49 -47 -6 -47 -6 51 -1 c46 -2
-53 -5 58 -26 9 -36 33 -56 69 -56 18 0 27 4 20 8 -6 4 -32 59 -58 122 -26 63
--50 118 -54 123 -4 4 -14 -8 -23 -27z" />
-            <path d="M538 324 c8 -4 12 -39 12 -124 0 -104 -2 -119 -17 -123 -10 -3 -2 -5
-17 -6 22 0 30 2 22 7 -9 6 -12 34 -10 103 l3 94 53 -104 c30 -57 57 -100 61
--95 17 18 28 69 19 79 -6 6 -29 46 -52 90 -40 79 -42 80 -81 82 -22 1 -34 0
--27 -3z" />
-            <path d="M778 318 c-19 -15 -36 -64 -19 -54 8 5 11 -21 11 -88 0 -66 -4 -96
--12 -99 -7 -3 10 -5 37 -5 28 0 44 2 38 5 -10 3 -13 38 -13 123 0 104 2 119
-18 123 9 3 3 5 -14 6 -16 0 -37 -5 -46 -11z" />
-            <path d="M888 324 c8 -4 12 -39 12 -124 0 -85 -4 -120 -12 -124 -7 -3 26 -5
-73 -3 102 2 132 19 127 71 -2 24 -11 35 -38 48 -19 10 -28 18 -19 18 21 0 49
-32 49 57 0 39 -34 56 -123 60 -45 1 -76 0 -69 -3z m120 -16 c28 -28 4 -98 -34
--98 -10 0 -14 14 -14 55 0 46 3 55 18 55 10 0 23 -5 30 -12z m3 -118 c14 -8
-19 -21 19 -48 0 -44 -14 -62 -47 -62 -21 0 -23 4 -23 60 0 64 9 72 51 50z" />
-            <path d="M1138 323 c9 -3 12 -38 12 -123 0 -85 -3 -120 -12 -123 -7 -3 15 -6
-50 -6 111 -2 162 39 162 129 0 90 -51 131 -162 129 -35 0 -57 -3 -50 -6z m128
--24 c15 -17 19 -36 19 -99 0 -83 -17 -120 -57 -120 -16 0 -18 10 -18 120 0
-110 2 120 18 120 11 0 27 -9 38 -21z" />
-            <path d="M1446 269 c-15 -34 -25 -64 -22 -66 2 -3 7 5 11 16 10 31 20 26 38
--19 l15 -40 -38 0 c-33 0 -38 3 -34 18 5 14 4 15 -5 2 -18 -26 -13 -30 38 -30
-43 0 50 -3 55 -24 9 -36 33 -56 69 -56 18 0 27 4 20 8 -6 4 -32 59 -58 122
--26 63 -50 119 -54 123 -5 4 -20 -20 -35 -54z" />
-            <path d="M736 235 c-9 -26 -7 -32 5 -12 6 10 9 21 6 23 -2 3 -7 -2 -11 -11z" />
-            <path d="M336 225 c-9 -26 -7 -32 5 -12 6 10 9 21 6 23 -2 3 -7 -2 -11 -11z" />
-            <path d="M716 185 c-9 -26 -7 -32 5 -12 6 10 9 21 6 23 -2 3 -7 -2 -11 -11z" />
-            <path d="M296 125 c-9 -26 -7 -32 5 -12 6 10 9 21 6 23 -2 3 -7 -2 -11 -11z" />
-            <path d="M1386 125 c-9 -26 -7 -32 5 -12 6 10 9 21 6 23 -2 3 -7 -2 -11 -11z" />
-            <path d="M269 79 c-10 -6 -4 -9 19 -9 17 0 32 2 32 4 0 9 -39 13 -51 5z" />
-            <path d="M1360 80 c-11 -7 -6 -10 20 -10 26 0 31 3 20 10 -8 5 -17 10 -20 10
--3 0 -12 -5 -20 -10z" />
-          </g>
-        </symbol>
-      </svg>
-      <svg class="logo">
-        <use xlink:href="#logo"></use>
-      </svg>
+     <a class="logo" hrev="#">Lambda</a>
```

Comment:
```suggestion
     <a class="logo" href="#">Lambda</a>
```

Reply:
fixed

Reply:
If so, where's the commit with the fix? 😄

Reply:
corrected